bears/c_languages/CPPCheckBear.py

```diff
@@ -42,4 +43,7 @@ def create_arguments(filename, file, config_file,
         if enable:
             args += ('--enable=' + ','.join(enable),)
 
+        if include_paths:
+            args += include_paths,
```

Comment:
This is a little confusing.
I assume that `include_paths` would be `-I` arguments, but here you add the paths without `-I`.

If you are adding directories at the end of the list of arguments, you are running `cppcheck` on all files in that directory, which mean it is a `GlobalBear`, not a `LocalBear` which works on one file at a time.